src/ssl/ErrorDetailManager.cc

```diff
@@ -258,7 +258,7 @@ Ssl::ErrorDetailFile::parse(const char *buffer, int len, bool eof)
 
         buf.consume(size);
     }
-    debugs(83, 9, HERE << " Remain size: " << buf.contentSize() << " Content: " << buf.content());
+    debugs(83, 9, "Remain size: " << buf.contentSize() << " Content: " << buf.content());
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.